mantidproject/mantid
# Add "Apply to All" buttons to the plot options sub-tabs

**Description of work.**
The Line, Marker, and Errorbars sub-tabs in the Curves tab of the figure options now have an "Apply to All" button which causes the properties of the current curve to be applied to all other curves in the plot. When the plot contains only one curve, the buttons are disabled.

**To test:**
1. In Workbench, plot multiple spectra on a single figure.
2. Open the figure options and go to the Curves tab.
3. For each of the Line, Marker, and Errorbars tabs, make some changes to the properties, click the Apply to All button, and the other curves should now have the same properties. Clicking the Apply button makes these changes visible on the figure.
4. Plot a new figure with a single spectrum and check that the Apply to All buttons are disabled.

Fixes #26591 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
